exonum/src/blockchain/schema.rs

```diff
@@ -384,40 +384,41 @@ enum IndexTag {
 /// Normalized coordinates of the index in the `state_hash_aggregator` table.
 #[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
 pub struct IndexCoordinates {
-    tag: u16,
-    group_id: u32,
+    origin: u16,
+    schema_id: u32,
     index_id: u16,
 }
 
 impl IndexCoordinates {
-    /// Creates index coordinated for the index with the specified owner and identifier.
-    pub fn new(owner: IndexOwner, index_id: u16) -> Self {
+    /// Creates index coordinates for the index with the specified schema origin
+    /// and index identifier.
+    pub fn new(schema_origin: SchemaOrigin, index_id: u16) -> Self {
         Self {
-            tag: owner.tag() as u16,
-            group_id: owner.group_id(),
+            origin: schema_origin.origin() as u16,
+            schema_id: schema_origin.schema_id(),
             index_id,
         }
     }
 
     // TODO Write more meaningful description [ECR-3824]
-    /// For the given index owner, returns a list of the index coordinates that match the corresponding
-    /// object hashes of the indices.
+    /// For the given index owner, returns a list of the index coordinates that match the
+    /// corresponding object hashes of the indices.
```

Comment:
if the "object hash" means the "hash of the index", I would delete the word "object" not to multiple different names for the same notion
```suggestion
    /// corresponding hashes of the indices.
```